lnd.go

```diff
@@ -368,9 +373,20 @@ func lndMain() error {
 	}
 	server.fundingMgr = fundingMgr
 
+	// Check macaroon authentication if macaroons aren't disabled.
+	if macaroonService != nil {
+		serverOpts = append(serverOpts,
+			grpc.UnaryInterceptor(macaroons.UnaryServerInterceptor(
+				macaroonService, permissions)),
+			grpc.StreamInterceptor(macaroons.StreamServerInterceptor(
+				macaroonService, permissions)),
+		)
+		fmt.Printf("Appended interceptor options\n")
```

Comment:
Is this left over from debugging sessions?

Reply:
Sure is. Deleted.